validation/static/record.js

```diff
@@ -0,0 +1,125 @@
+"use strict";
+
+// referencing: https://blog.addpipe.com/using-recorder-js-to-capture-wav-audio-in-your-html5-web-site/
+
+let gumStream;      // stream from getUserMedia()
+let rec;            // Recorder.js object
+let input;          // MediaStreamAudioSourceNode
+const AudioContext = window.AudioContext;
+let audioContext = new AudioContext;
```

Comment:
Does it still work if you do:

```suggestion
const audioContext = new AudioContext();
```

?

Reply:
Sure does!